Using-Mastodon/FAQ.md

```diff
@@ -64,75 +71,147 @@ Try [TootyFruity](https://play.google.com/store/apps/details?id=ch.kevinegli.too
+<img src="screenshots/admin_info.png" alt="Admin Info" height="200"/>
 
 #### Wow, I keep seeing offensive stuff from a particular instance, is there a way to block *all* of it?
 Not without a lot of clicking, sorry. But, your instance admin can do it! Please contact her.
 
+#### I blocked someone, why can I still see their posts?
+Blocking doesn’t catch when toots are boosted.
+
+#### Is Mastodon moderated?
+Each instance handles moderation differently, and each has their own moderators. It’s okay to ask what the rules are for the instance you are on. Usually the rules are posted on the instance's about page ([like the “more” page at Mastodon.social](https://mastodon.social/about/more))
+
+#### How do I report offensive content?
+Beneath each post, you will see three dots. Clicking on those will allow you to expand a post or report the content. When reporting content, select all posts that need moderator attention.
+
+#### How do I handle harassment?
+Harassment is taken seriously by the admins and moderators of instances and should be reported through the post-reporting system which can be found via the three dots under every post or writing your admin/mod directly.
+
+#### Do my Private Messages get to people on other instances?
+They should, yes. However, because of different privacy settings and other factors, your post might not remain private under certain circumstances.
+
+#### What does “Adjust Status Privacy” mean and how does it work?
+The **globe** icon under the toot area adjusts your status privacy by changing who can see your posts. This is what happens:
+
+| Privacy setting | Destination | Notes |
+| --------------- | ----------- | ----- |
+| Public          | global, public timeline | Federated. People on other instances will probably see it. |
+| Unlisted        | local public timeline | Can be boosted and thus seen on other instances.  |
+| Private         | only to your followers |  Cannot be boosted. |
+| Direct          | mentioned people only  | Similar to email |
+
+#### Can I use hashtags? Should I?
+Yes! Hashtags are tracked and are often fun, but some tags help people avoid triggering posts. This is especially appreciated on public posts of #POLITICS, #HEALTH, #DEPRESSION, or #LEWD OR #NSFW topics. Such posts are also what the Content Warning system was designed for.
+
 #### How do I get verified with a “✅”?
 “✅” is an emoji, *only* for lulz. There is no verification on Mastodon. You can copy and paste “✅” into your bio if you wish, you cheeky monkey.
 
 #### Can I edit a toot?
-No, sorry. But you could delete and rewrite...
+No, sorry. But you could delete your toot and rewrite…
+
+#### If I delete a post, does it get deleted everywhere?
+Deleting a post will delete it from the local timeline. However, if it has already been boosted, it may still be seen and may federate.
 
 #### How do I view my favorite toots?
-Click "Getting Started," then "Favourites."
+Click **Getting Started**, then **Favourites**.
```

Comment:
Yes, changing it is clunky. Either it's inaccurate or the style of the whole FAQ should be in UK English. Maybe a little mention: "Click **Getting Started**, then **Favourites**. Yes—Mastodon uses UK spelling!"